library/spdm_responder_lib/libspdm_rsp_encap_get_certificate.c

```diff
@@ -8,6 +8,7 @@
 
 #if LIBSPDM_ENABLE_CAPABILITY_CERT_CAP
 
+uint16_t total_responder_cert_chain_buffer_length;
```

Comment:
We cannot use global variable.

Reply:
Yes. We should observe the README. I have updated the code.